scripts/gen_og.py

```
@@ -0,0 +1,115 @@
+BORDER = "#2C2C2E"
+
+img = Image.new("RGB", (W, H), BG)
+draw = ImageDraw.Draw(img)
+
+try:
+    font_bold = ImageFont.truetype("/usr/share/fonts/TTF/DejaVuSans-Bold.ttf", 64)
+    font_mono = ImageFont.truetype("/usr/share/fonts/TTF/DejaVuSansMono-Bold.ttf", 24)
+    font_mono_sm = ImageFont.truetype("/usr/share/fonts/TTF/DejaVuSansMono.ttf", 18)
+    font_sans = ImageFont.truetype("/usr/share/fonts/TTF/DejaVuSans.ttf", 28)
+    font_sans_sm = ImageFont.truetype("/usr/share/fonts/TTF/DejaVuSans.ttf", 20)
+except:
```

Ruff (0.15.17)

[error] Do not use bare `except`

(E722)